content/blog/how-to-develop-a-serverless-application-with-fission-pt-1.md

```diff
@@ -0,0 +1,295 @@
++++
+type = "post"
+author = "ta-ching chen"
+title = "How to Develop a Serverless Application with Fission (Part 1)"
+description = ""
+spliter = " | "
+categories = [""]
+tags = [""]
+keywords = ""
+date = "2018-08-07T15:48:12+08:00"
+
+draft = true
+
+images = ["/img/fission-and-istio-integration/fission.min.png"]
+featured = "fission.min.png"
+featuredpath = "/img/fission-and-istio-integration"
+featuredalt = ""
+linktitle = ""
++++
+
+# Introduction
+
+When users try to develop serverless applications, they often choose some exisiting and mature solutions like [AWS Lambda](https://aws.amazon.com/lambda/) or [Google Cloud Functions](https://cloud.google.com/functions/). However, users may encounter following problems after application development:
+
+1. It's hard to simulate the same environment on local machine. 
+2. Vendor lock-in problem make it hard to move applications from one cloud provider to another.
+3. Some sensitive data may not allow to send outside of on-premises datacenter.
+
+[Fission](https://fission.io/) is a serverless framework built on top of [Kubernetes](https://kubernetes.io/). Here are some benefits for adopting Fission as serverless solution to develop a severless application:
+
+* `Portability`
+    * Labtop, PC, On-premises datacenter and even cloud providers wherever you can run kubernetes, you can run Fission.
```

Comment:
* Laptop, PC, On-premises datacenter, even cloud providers. Wherever you can run Kubernetes, ...